src/main/java/edu/ie3/datamodel/models/input/connector/Transformer3WInput.java

```diff
@@ -186,6 +228,10 @@ public NodeInput getNodeInternal() {
     return nodeInternal;
   }
 
+  public Transformer3WInputCopyBuilder copy() {
```

Comment:
```suggestion
  public Transformer3WInputCopyBuilder getCopyBuilder() {
```

From my point of view this thing is not to vital. I was just wondering, if you would expect return type `Transformer3WInput` and getting a clean copy, instead of a builder. Especially, as no doc is given, what this method is meant to do.

Additionally, we should spend some thought on, if the `copy`-method shouldn't be part of abstract class `AssetInput`?

Reply:
I would prefer stay with `copy()` as the return type is clear by type safety. Furthermore having `copy().build()` for an unmodified copy is IMHO very expressive.